### DISCUSSION TITLE
希望MP-CU拥有哪些功能？来说说，说不定就有了呢

### DISCUSSION THREAD

**Discussion by iZaiZaiA** [Color-UI/MP-CU#19] [Ideas]
您希望MP-CU增加些什么功能？ 或者，您现在的项目急需的组件，而MP-CU又没有的呢？

大胆的提出您的想法或建议，说不定，下一次更新，就有您需要的组件或功能了呢 :)

**Comment by iibrat** [comment]
希望赶紧完成现有功能😸
